crates/cargo-test-macro/src/lib.rs

```diff
@@ -187,7 +187,7 @@ fn to_token_stream(code: &str) -> TokenStream {
 
 static mut VERSION: (u32, bool) = (0, false);
 
-fn version() -> &'static (u32, bool) {
+fn version() -> (u32, bool) {
```

Comment:
This seems to be unrelated to our changes. Why did this change become necessary?

Reply:
I removed the changes, but the CI will fail. Should we disable rust nightly and beta tests?